hack/ci-e2e.sh

```diff
@@ -54,11 +54,33 @@ docker exec vbmc vbmc list
 # These variables are used by the tests. They override variables in the config file.
 # This IP is defined by the network we created above.
 # Together with the VBMC_PORT this becomes the BMC_ADDRESS used by the BMH in the test.
-export BMC_ADDRESS="ipmi://192.168.222.1:${VBMC_PORT}"
+IP_ADDRESS="192.168.222.1"
+export BMC_ADDRESS="ipmi://${IP_ADDRESS}:${VBMC_PORT}"
 export BOOT_MAC_ADDRESS
 # These are the VBMC defaults (used since we did not specify anything else for `vbmc add`).
 export BMC_USER=admin
 export BMC_PASSWORD=password
+IMAGE_FILE="cirros-0.6.2-x86_64-disk.img"
```

Comment:
This looks really great! Thank you a lot xD
I'm wondering though if we should put the cirros img version into a variable, for future upgrade? This `0.6.2` does appear again and again

Reply:
Also, maybe we should change `IMAGE_FILE` to something more specific? Like `CIRROS_IMAGE_FILE` or `IMAGE_SERVER_E2E_FILE` or something?

Reply:
Good point about the version. I don't have a strong opinion on it but it would sure simplify a bit when changing it later.

Regarding `IMAGE_FILE` I think we should not change it. We happen to use cirros now, but there is nothing preventing us from using other images in the future so I don't think it should include the OS. Perhaps there would be some benefit to `DISK_IMAGE_FILE` or similar, to make it obvious what kind of image it is? But honestly to me it seems good like it is already. The context in the script should explain it pretty good :slightly_smiling_face:

Reply:
I could do a variable for the cirros version if you like but then we need to download the MD5 file and do a checksum test aswell. I'll dig into that and will also change the name to ``IMAGE_FILE_E2E``.  Thanks for the review!

Reply:
Wait wait, please don't change it to `IMAGE_FILE_E2E`. The context should already be clear. The whole file is about e2e so no need to name the variables e2e.
Good point about the MD5 also. I don't think we should download the MD5 file. It is just unnecessarily complicating things at this point. The only issue (which is really small I would say) is that we have the version (0.6.2) in multiple places, which makes it slightly harder to update. If you want to improve this, you can create a variable for it (e.g. `CIRROS_VERSION`). It can then be used in the file name and URL. Updating the image would then only require changing this variable and the MD5 sum.

Reply:
Got it. Good points.